packages/documentation/src/stories/components/textarea/textarea.docs.mdx

```diff
@@ -0,0 +1,102 @@
+import { Meta, Canvas, Story, Source, ArgsTable } from '@storybook/addon-docs';
+
+<Meta title="Components/Textarea" />
+
+# Textarea
+
+<p class="lead">
+  Customize the native <code>&lt;textarea&gt;</code> with CSS that changes the element’s initial
+  appearance.
+</p>
+
+Custom `textarea` elements need only the class `form-control` to trigger the custom styles.
+
+<Canvas>
+  <Story id="components-textarea--default" />
+</Canvas>
+<ArgsTable story="^" />
+
+## SCSS Imports
+
+To import all styles from the design-system-styles package:
+
+```css dark
+@use "@swisspost/design-system-styles/index.scss";
+```
+
+To import only the styles required for this component:
+
+```css dark
+@use "@swisspost/design-system-styles/basis.scss";
+@use '@swisspost/design-system-styles/components/forms.scss';
+
+// required if you use floating-labels
+@use '@swisspost/design-system-styles/components/floating-label.scss';
+
+// required if you use validation feedbacks
+@use '@swisspost/design-system-styles/components/form-feedback.scss';
+```
+
+## Specific examples of application
+
+The following examples show the different characteristics of the component. These can differ in the type of visualization, the html structure, as well as when, how and why they are displayed.
+
+### Floating Label
+
+Wrap a pair of `textarea` and `label` elements in a `.form-floating` container to enable floating labels.<br/>
+But note that the `textarea` must come first, so we can ensure the correct styles.
+
+<Canvas>
+  <Story id="components-textarea--floating-label" />
+</Canvas>
+
+### Sizing
+
+The size can be changed by simply adding one of four classes:
+
+- Small (`form-control-sm`)
+- Regular (`form-control-rg`)
+- Medium (default or `form-control-md`)
+- Large (`form-control-lg`)
+
+<p class="text-danger">
+  Note: Do not use this classes on a <code>textarea</code> element wrapped within a
+  <code>.floating-label</code> container.
+</p>
+
+<Canvas>
+  <Story id="components-textarea--size" />
+</Canvas>
+
+### Rows
+
+To set a custom height on your `textarea`, use the `rows` attribute.
+
+<Canvas>
+  <Story id="components-textarea--rows" />
+</Canvas>
+
+### Helper Text
+
+Optional descriptive text which is always shown in a `.form-text` element.
+
+<Canvas>
+  <Story id="components-textarea--helper-text" />
+</Canvas>
+
+### Disabled
+
+Add the disabled boolean attribute on a `textarea` to give it a grayed out appearance and remove pointer events.
```

Comment:
Disabled controls should be avoided whenever possible because of accessibility. Our general stance is: Enable all controls and allow the user to submit a form. If there is invalid input, display an error message that enables the user to correct the issue. A disabled element cannot be focused and screenreaders cannot access it.

Should we add an info box that states:
"Disabled elements are not accessible. Avoid disabled elements wherever possible. If the element is not editable, display the value as text (`<p>`). If the user needs to correct the input, display an error message that helps the user fix the error. If the field is irrelevant in the current context, hide it entirely."